packages/dev/core/src/Meshes/geometry.ts

```diff
@@ -1204,7 +1204,7 @@ export class Geometry implements IGetSetVerticesData {
 
         if (this.isVerticesDataPresent(VertexBuffer.MatricesIndicesKind)) {
             serializationObject.matricesIndices = this._toNumberArray(this.getVerticesData(VertexBuffer.MatricesIndicesKind));
-            serializationObject.matricesIndices._isExpanded = true;
+            serializationObject.matricesIndicesExpanded = true;

```

Comment:
I think this is not the only property that has this problem. We should maybe fix all of them?